README

```diff
@@ -14,3 +14,15 @@ $ make
 $ make DD=/path/to/directory install
 
 Running against non-installed tree (except for the in-tree testsuite) is not supported.
+
+
+To install directly from GitHub use:
+
+$ curl -o- https://raw.githubusercontent.com/beakerlib/beakerlib/master/install.sh | bash
```

Comment:
I think https://github.com/beakerlib/beakerlib/raw/master/install.sh might be better.

Reply:
That results in 404 so no.